modules/nf-core/gcta/gsmr/main.nf

```diff
@@ -0,0 +1,63 @@
+process GCTA_GSMR {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gcta:1.94.1--h9ee0642_0':
+        'biocontainers/gcta:1.94.1--h9ee0642_0' }"
+
+    input:
+
+    tuple val(meta) , path(exposure)
+    tuple val(meta2), path(outcome)
+    path(reference)
+
+    output:
+    path "${meta.id}_${meta2.id}.log"         , emit: log
+    path "${meta.id}_${meta2.id}.gsmr"        , emit: gsmr
+    path "${meta.id}_${meta2.id}.eff_plot.gz" , emit: eff_plot, optional: true
+    path "${meta.id}_${meta2.id}.mono.badsnps", emit: mono_badsnps, optional: true
+    path "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
```

Comment:
`args` are defined but not used. If there are any other currently not used args, `args` must be in the script.